building/gits/optee_test.rst

```diff
@@ -193,6 +193,15 @@ Coding standards
 ****************
 See :ref:`coding_standards`.
 
+The :ref:`optee_os` repository is required to run the checks. It's location may
+be passed using the `OPTEE_OS_SRC` environment variable:
```

Comment:
Wondering if `OPTEE_OS_PATH` might be more appropriate? Given that we have: https://github.com/OP-TEE/build/blob/3.8.0/common.mk#L14.

Reply:
Yes, why not. Updated both PRs.